.travis.yml

```diff
@@ -6,6 +6,7 @@ addons:
     packages:
       - expect-dev  # provides unbuffer utility
       - python-lxml # because pip installation is slow
+      - python-pycountry
```

Comment:
It can be in both places, but I'll do in requirements.txt to specify the minimum version where historic countries has been introduced